### DISCUSSION TITLE
97 Things Every Programmer Should Know: Collective Wisdom from the Experts

### DISCUSSION THREAD

**Discussion by lincolixavier** [pilhacheia/clube-do-livro-dev#19] [Books]
Achei muito curioso! Deve ser uma leitura bem legal!

 [Link da amazon](https://amzn.to/3wlE9qt)

**Comment by hkupty** [comment]
Já li (quase) todo. Muito bom. Várias pérolas de conhecimento. Recomendo ler um "capítulo" por dia.
